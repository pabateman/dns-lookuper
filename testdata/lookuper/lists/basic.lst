# The comments
iana.org # is
kernel.org #every
# where foo.kernel.org
