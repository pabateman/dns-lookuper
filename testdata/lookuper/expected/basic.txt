139.178.84.217
192.0.43.8
